lib/ex_quiz_it/questions/enums.ex

```diff
@@ -0,0 +1,38 @@
+defmodule ExQuizIt.Questions.Enums do
+  @moduledoc false
+  def q_and_a do
```

Comment:
Maybe `questions_and_answers`?

Reply:
not sure. i like that this is short. so I will leave it for now